test/extensions/filters/network/redis_proxy/command_splitter_impl_test.cc

```diff
@@ -769,6 +770,47 @@ INSTANTIATE_TEST_SUITE_P(
     RedisSplitKeysSumResultHandlerTest, RedisSplitKeysSumResultHandlerTest,
     testing::ValuesIn(Common::Redis::SupportedCommands::hashMultipleSumResultCommands()));
 
+class RedisSingleServerRequestWithLatencyMicrosTest : public RedisSingleServerRequestTest {
+public:
+  void makeRequest(const std::string& hash_key, const Common::Redis::RespValue& request) {
+    EXPECT_CALL(*conn_pool_, makeRequest(hash_key, Ref(request), _))
+        .WillOnce(DoAll(WithArg<2>(SaveArgAddress(&pool_callbacks_)), Return(&pool_request_)));
+    handle_ = splitter_.makeRequest(request, callbacks_);
+  }
+
+  ConnPool::MockInstance* conn_pool_{new ConnPool::MockInstance()};
+  InstanceImpl splitter_{ConnPool::InstancePtr{conn_pool_}, store_, "redis.foo.", time_system_,
+                         true};
```

Comment:
Can we have another test that verifies the behavior with `latency_in_micros` set to `false`?

Reply:
The other tests in the file are setting the latency_in_micros to false.

Reply:
Fair enough.